example/src/main/java/com/stripe/example/activity/NetbankingPaymentActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.stripe.example.activity
+import android.os.Bundle
+import android.view.View
+import android.widget.ArrayAdapter
+import androidx.lifecycle.Observer
+import com.stripe.android.model.Address
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.example.databinding.NetbankingPaymentActivityBinding
+
+class NetbankingPaymentActivity : StripeIntentActivity() {
+    private val viewBinding: NetbankingPaymentActivityBinding by lazy {
+        NetbankingPaymentActivityBinding.inflate(layoutInflater)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(viewBinding.root)
+
+        viewModel.inProgress.observe(this, { enableUi(!it) })
+        viewModel.status.observe(this, Observer(viewBinding.status::setText))
+
+        val adapter = ArrayAdapter(this, android.R.layout.expandable_list_content, arrayListOf("hdfc", "icici", "sbi", "axis", "hdfc_fake"))
```

Comment:
we have a list of around ~120 banks. Not sure what is the best way to store these values. Should I maintain it as a constant in strings.xml?

Reply:
Is it possible to have a dropdown along with type suggestions? What is the suggested value to use instead of android.R.layout.expandable_list_content

Reply:
Let's use a `Spinner`